types/interfaces.go

```diff
@@ -71,4 +71,5 @@ type Pool interface {
 	Switch(API, ClientMessage) error
 	Save()
 	Add(string, uuid.UUID, map[Socket]bool) (API, error)
+	GetImageDB() interface{}
```

Comment:
Could this function return an `ImageDB` type, instead of a generic `interface{}`, so we don't have to do any casting later?

Reply:
Nope, import loop :(

Reply:
Types cannot depend on something on pool, although perhaps we could move all the ImageDB stuff into types?

Reply:
Does the `types` package really import the `pool` package? It shouldn't since that's the whole point of the types package: everything imports `type`, but `type` imports none of our packages